src/components/MapCanvas.js

```diff
@@ -1,27 +1,62 @@
 import React, { Component } from 'react';
-import { Header } from 'semantic-ui-react';
+import ReactMapGL, { Marker } from 'react-map-gl';
+import styled from 'styled-components/macro';
+import LocationIcon from '@material-ui/icons/Lens';
 
 import summary from '../apis/summary';
 
+const MapWrapper = styled('div')`
+  position: absolute;
+`;
+
 class MapCanvas extends Component {
   state = {
-    geometry: {}
+    viewport: {
+      width: '100vw',
+      height: '100vh',
+      latitude: -12.287,
+      longitude: 113.278,
+      zoom: 3
+    },
+    geoObject: null
   };
 
   componentDidMount() {
     this.onViewChange();
   }
 
   async onViewChange() {
-    const geoData = {
-      geometry: this.state.geometry
-    };
-    const { data: result } = await summary.post('/sites', geoData);
-    console.log(result);
+    const { data: result } = await summary.post('/sites/');
+    this.setState({ geoObject: result });
   }
 
   render() {
-    return <Header as="h1">MAP</Header>;
+    const siteLocations = this.state.geoObject ? (
+      this.state.geoObject.features.map(site => (
+        <Marker
+          key={site.id}
+          latitude={site.geometry.coordinates[1]}
+          longitude={site.geometry.coordinates[0]}
+        >
+          <LocationIcon color="secondary" />
+        </Marker>
+      ))
+    ) : (
+      <div>NOT FOUND</div>
+    );
+
+    return (
+      <MapWrapper>
+        <ReactMapGL
+          {...this.state.viewport}
+          mapboxApiAccessToken={process.env.REACT_APP_MAPBOX_TOKEN}
+          mapStyle="mapbox://styles/nickhoang/cjx6prmf703ev1cqz9qd1ykdr"
```

Comment:
As a future ticket, you should make sure this style is owned and maintained on a WCS account, and not your personal. You should do this whenever you're confident that the underlying style is production-ready :)

Reply:
Captured in #9